packages/ringcentral-widgets-test/test/integration-test/AllCallsPage/AllCalls.spec.js

```diff
@@ -0,0 +1,145 @@
+import * as mock from 'ringcentral-integration/integration-test/mock';
+import IncomingCallPanel from 'ringcentral-widgets/components/IncomingCallPanel';
+import webphoneErrors from 'ringcentral-integration/modules/Webphone/webphoneErrors';
+import NavigationBar from 'ringcentral-widgets/components/NavigationBar';
+import TabContentPanel from 'ringcentral-widgets/components/TabContentPanel';
+import TabNavigationButton from 'ringcentral-widgets/components/TabNavigationButton';
+import ActiveCallList from 'ringcentral-widgets/components/ActiveCallList';
+import ActiveCallItemV2 from 'ringcentral-widgets/components/ActiveCallItemV2';
+import ActiveCallItem from 'ringcentral-widgets/components/ActiveCallItem';
+import telephonyStatus from 'ringcentral-integration/enums/telephonyStatuses';
+import CircleButton from 'ringcentral-widgets/components/CircleButton';
+import { getWrapper, timeout } from '../shared';
```

Comment:
plz replace getWrapper to initPhoneWrapper avoiding different tests interfering with each other